system/doc/reference_manual/expressions.xml

```diff
@@ -836,6 +836,21 @@ number &lt; atom &lt; reference &lt; fun &lt; port &lt; pid &lt; tuple &lt; map
       depending on the size of the float because otherwise comparison of large
       floats and integers would lose their transitivity.</p>
 
+    <p>The term equivalence operators, <c>=:=</c> and <c>=/=</c>, return
+      whether two terms are indistinguishable. While the other operators
+      consider the same <em>numbers</em> equal even when their types differ
+      (<c>1 == 1.0</c> is true), the term equivalence operators return whether
+      there exists any function that can tell their arguments apart.</p>
+    <p>For example, while the terms <c>0.0</c> and <c>-0.0</c> represent the
+      exact same <em>number</em>, they yield different results when converted
```

Comment:
I think we need a *since* note of some kind here since that distinction is new.

Reply:
Good point! :)

Reply:
We should also explain that `0.0` and `-0.0` are distinct terms here:
https://www.erlang.org/doc/reference_manual/data_types#representation-of-floating-point-numbers
with a note when that changed...

Reply:
Perhaps a good place to state that `0.0` and `+0.0` are equivalent